File: 561-array-partition/array-partition.java
class Solution {
    public int arrayPairSum(int[] nums) {
        Arrays.sort(nums);
        int total_sum = 0 ;
        for ( int i =0 ; i < nums.length ; i+=2 ){
            total_sum += nums[i];


        }
        return total_sum;
    }
}

File: 561-array-partition/array-partition.py
class Solution:
    def arrayPairSum(self, nums: List[int]) -> int:
        nums.sort()
        
        # Step 2: Initialize the sum
        total_sum = 0
        
        # Step 3: Sum up every second element (the minimums of pairs)
        for i in range(0, len(nums), 2):
            total_sum += nums[i]
        
        return total_sum
